### PR TITLE
Add Increment and Decrement Command

### LINKED ISSUE #12: Add "inc" command (items) to increment item quantity
This command is a new command.

It requires the inventory item model to be implemented first.

An alternative way to change the quantity of the item is via the "set" command.
 "item set qty/10".
https://github.com/AY2223S1-CS2103T-W16-2/tp/issues/11 

The "inc" command is a multi step command related to items.

End result:
e.g: "item inc 1 10"

Implementation should be similar to : https://github.com/AY2223S1-CS2103T-W16-2/tp/issues/13

### PR DESCRIPTION
**Pull request by Ferusel**
Closes #12 and #13.

Adds the increment and decrement command.

### CODE REVIEW

File: src/main/java/seedu/foodrem/logic/parser/DecrementCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_ITEM_QUANTITY);
+        Index index;
+
+        try {
+            index = ParserUtil.parseIndex(argMultimap.getPreamble());
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditCommand.MESSAGE_USAGE), pe);
+        }
+
+        ItemQuantity decrementQuantity = ParserUtil.parseQuantity(argMultimap.getPresentValue(PREFIX_ITEM_QUANTITY));
```
[the remark this answers is not in the document]
Comment (yixiann) [reply src/main/java/seedu/foodrem/logic/parser/DecrementCommandParser.java:34]: I am not sure what is the behaviour if you do not check

[the remark this answers is not in the document]
Comment (Ferusel) [reply src/main/java/seedu/foodrem/logic/parser/DecrementCommandParser.java:34]: Thanks! I wasn't sure how to check if the prefix is missing.

**Review by yixiann — DISMISSED** [review]
I prefer this implementation but should see what Bryan and RIchard thinks

**Review by RichDom2185 — APPROVED** [review]
LGTM

**Pull request merged**